back/app/models/permission.rb

```diff
@@ -25,6 +25,7 @@ class Permission < ApplicationRecord
     nil => %w[visiting following posting_initiative commenting_initiative reacting_initiative],
     'information' => [],
     'ideation' => %w[posting_idea commenting_idea reacting_idea],
+    'proposals' => %w[posting_idea commenting_idea reacting_idea],
```

Comment:
Again, just thinking in public here, but would probably be more flexible if we drop the `_idea` and other method references in these action names.

Reply:
I would love to drop the `_idea` part. In general, I think there are a lot of occurrences of `idea` throughout the code that we could either drop or rename to `input`. We could rename the ideas table and model, rename relationships like `idea_images` and `idea_statuses`, and there's probably more. @IvaKop also mentioned today that getting back `type: 'idea'` in the API responses for proposals and survey responses is confusing.

@kogre perhaps we could make a separate ticket for renaming occurrences of `idea`  with a slightly lower priority?

Reply:
Yes, agreed, we should bite that bullet at some point, so let's be positive and add it (low) on our list

Reply:
Added a ticket here: https://www.notion.so/govocal/Occurrences-of-idea-Drop-or-rename-to-input-b9ad4f956dd44bdfb20c0ac4fd5a1069?pvs=4